Makefile

```diff
@@ -25,6 +25,8 @@ IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:crdVersions=v1"
 
+SHELL=/bin/bash
```

Comment:
Observing `/bin/sh: 1: Syntax error: redirection unexpected` as here-strings are used in verify-go-version.
 Hence, setting shell to bash shell from user shell as a workaround.